orbit-2-rxjava2/src/main/java/com/babylon/orbit2/RxJava2Single.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2020 Babylon Partners Limited
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.babylon.orbit2
+
+import io.reactivex.Single
+
+internal class RxJava2Single<S : Any, E : Any, E2 : Any>(val block: suspend Context<S, E>.() -> Single<E2>) :
+    Operator<S, E>
```

Comment:
feels a little pedantic but would this formatting read better?
```suggestion
internal class RxJava2Single<S : Any, E : Any, E2 : Any>(
    val block: suspend Context<S, E>.() -> Single<E2>
) : Operator<S, E>
```